docs/streams/developer-guide.html

```diff
@@ -1828,8 +1837,10 @@
             () -> 0, /* initializer */
             (aggKey, newValue, aggValue) -> aggValue + newValue, /* adder */
             (aggKey, oldValue, aggValue) -> aggValue - oldValue, /* subtractor */
-            Serdes.Integer(), /* serde for aggregate value */
-            "aggregated-table-store" /* state store name */);
+            Materialized.&lt;String, Integer, KeyValueStore&lt;Bytes, byte[]&gt;&gt;as("aggregated-table-store")
+                .withKeySerde(Serdes.String() /* serde for aggregate key */)
```

Comment:
why do we add this? was the example wrong ?

Reply:
otherwise it will use the default key serde. And seeing as it is not specified above we should set it here.